lib/multiplexer/multiplexer.go

```diff
@@ -94,17 +95,33 @@ func New(cfg Config) (*Mux, error) {
 		return nil, trace.Wrap(err)
 	}
 
+	entry := log.WithFields(log.Fields{
+		trace.Component: teleport.Component("mx", cfg.ID),
+	})
+	logLimiter, err := loglimit.New(loglimit.Config{
+		Entry:         entry,
```

Comment:
Legit breakage?

Reply:
Thanks! Fixed in 58cb1b668c53c55fdf24f19fbb3bde6724e41924.